GitUI/UserControls/RevisionGrid/Columns/CommitIdColumnProvider.cs

```diff
@@ -35,12 +36,12 @@ public CommitIdColumnProvider(RevisionGridControl grid)
 
         public override void OnCellPainting(DataGridViewCellPaintingEventArgs e, GitRevision revision, int rowHeight, in (Brush backBrush, Color foreColor, Font normalFont, Font boldFont) style)
         {
-            if (!_widthByLengthByFont.TryGetValue(style.normalFont, out var widthByLength))
+            if (!_widthByLengthByFont.TryGetValue(_monospaceFont, out var widthByLength))
```

Comment:
we need to update the method signature to pass the monospace font, e.g.

```
public override void OnCellPainting(DataGridViewCellPaintingEventArgs e, GitRevision revision, int rowHeight, 
         in (Brush backBrush, Color foreColor, Font normalFont, Font boldFont, Font monospaceFont) style)
```

@drewnoakes is there a reason why `style` isn't a defined struct? e.g.
```
public struct RowStyle 
{
    ...
}
```

Reply:
It could be converted to a struct. I'd suggest it be made `public readonly struct CellStyle` if that name doesn't conflict. `readonly` structs aren't subject to defensive copies when used as `in` parameters.

Reply:
Moved style stuff into `CellStyle`. Wish I had made this decision myself. It seemed excessive to change the ValueTuple in all those places just so that `CommitIdColumnProvider` could use it, makes sense with a dedicated style struct.